### DISCUSSION TITLE
Bulk script save location?

### DISCUSSION THREAD

**Discussion by BananamanCoolDude** [amidaware/tacticalrmm#1231] [Q&A] [answered]
Good morning Guys,

I was wondering where does RMM save bulk script text outputs for example for BitLocker keys?

![image](https://user-images.githubusercontent.com/102809616/181745578-7f7e6262-c94a-4ceb-befb-13c153fbe777.png)

**Comment by dinger1986** [comment]
I dont think it does

**Comment by silversword411** [comment, marked as answer]
It's in the history of any agent you've run it against. 

Unfortunately can't "save output to notes" with the bulk script tool ;)
